.github/ISSUE_TEMPLATE/other_bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Other Bug Report
+about: Create a bug report for other problems with Adaptive Crds
```

Comment:
My $.02 is that this one isn't necessary.

Reply:
I went back and forth about that...I was trying to have a "catch-all" bucket to prevent people opening bugs in certain areas when they were unsure but I don't know if it is really necessary. I can remove it, I don't feel strongly either way